### PR TITLE
Filter entry list by lead title v2

### PR DESCRIPTION
**Pull request by annshress**

### CODE REVIEW

File: apps/entry/filter_set.py [hunk 1]
```diff
@@ -143,6 +148,13 @@ class Meta:
             },
         }
 
+    # def lead_title_filter(self, queryset, name, value):
+    #     if value.strip():
+    #         return queryset.annotate(
+    #             idx=StrIndex(Lower('lead__title'), Value(value.lower()))
+    #         ).filter(idx__gt=0).order_by('idx', 'lead__title')
+    #     return queryset
+
```
Comment (thenav56) [inline apps/entry/filter_set.py:157]: Let's not keep commented code. If you think it's good to keep this, let's create a 2 commit w/o this.

File: apps/entry/filter_set.py [hunk 2]
```diff
@@ -120,13 +121,17 @@ class EntryFilterSet(django_filters.rest_framework.FilterSet):
         label='Lead Group Label',
         method='lead_group_label_filter',
     )
+    lead__title = django_filters.CharFilter(
+        field_name='lead__title',
+        lookup_expr='icontains'
+    )
```
Comment (thenav56) [inline apps/entry/filter_set.py:127]: Let's add a test for this.

File: apps/entry/filter_set.py (hunk 1 above)
Comment (tnagorra) [reply apps/entry/filter_set.py:157]: I second @thenav56

**Comment by AdityaKhatri** [comment]
@annshress , lets close this.

**Pull request closed without merging**